### PR TITLE
Allow running in a directory other than .

### PR DESCRIPTION
**Pull request by 2opremio**
Add an input parameter `working-directory` which allows setting the directory to run on

### CODE REVIEW

**Comment by chetan** [comment]
Does setting the working-directory on the job itself not work?

```
    - name: restore timestamps
      uses: chetan/git-restore-mtime-action@v2
      working-directory: ~/foo/bar
```

File: action.yml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 name: 'git-restore-mtime'
 description: "Restore file's modified time (timestamp) based on git commits"
 author: 'Pixelcop Research, Inc.'
+inputs:
+  working-directory,:
```
Comment (chetan) [inline action.yml]: Errant comma?

**Comment by 2opremio** [comment]
I am using it in a compose action where I don’t think you can set the working directory

File: action.yml (hunk 1 above)
Comment (2opremio) [reply action.yml]:
```suggestion
  working-directory:
```

**Review by chetan — APPROVED** [review]

**Comment by chetan** [comment]
@2opremio do you have an example of how you're using it in your composite action? I want to update the readme to show the new option. Would it be enough to just show an example and document it? i.e., add

```yaml
with:
  working-directory: /foo/bar
```

and describe the param as optional, etc

**Comment by 2opremio** [comment]
Thanks for merging.

Here is an example of how the new parameter works: https://github.com/stellar/soroban-rpc/pull/229/files

**Comment by chetan** [comment]
Thanks! I just cut the v2.1 tag and updated v2

**Pull request merged**